openmetadata-ui/src/main/resources/ui/src/components/DataQuality/TestSuite/TestSuiteList/TestSuites.component.tsx

```diff
@@ -276,88 +299,91 @@ export const TestSuites = () => {
+                  value={ownerFilterValue}
+                />
+              </UserTeamSelectableList>
+            </Form.Item>
+          </Space>
+        </Form>
       </Col>
 
       <Col span={24}>
-        <SummaryPanel
-          showAdditionalSummary
+        <PieChartSummaryPanel
           isLoading={isTestCaseSummaryLoading}
           testSummary={testCaseSummary}
         />
       </Col>
+
       <Col span={24}>
-        <Table
-          columns={columns}
-          customPaginationProps={{
-            currentPage,
-            isLoading,
-            pageSize,
-            isNumberBased: true,
-            paging,
-            pagingHandler: handleTestSuitesPageChange,
-            onShowSizeChange: handlePageSizeChange,
-            showPagination,
-          }}
-          data-testid="test-suite-table"
-          dataSource={testSuites}
-          loading={isLoading}
-          locale={{
-            emptyText: <FilterTablePlaceHolder />,
-          }}
-          pagination={false}
-          scroll={{
-            x: '100%',
-          }}
-          size="small"
-        />
+        <div className="test-suite-list-container">
+          <div className="test-suite-list-header">
+            <Row gutter={[16, 16]}>
+              <Col data-testid="test-suite-sub-tab-container" span={16}>
+                <Radio.Group value={subTab} onChange={handleSubTabChange}>
+                  <Radio.Button
+                    data-testid="table-suite-radio-btn"
+                    value={DataQualitySubTabs.TABLE_SUITES}>
+                    {t('label.table-suite-plural')}
+                  </Radio.Button>
+                  <Radio.Button
+                    data-testid="bundle-suite-radio-btn"
+                    value={DataQualitySubTabs.BUNDLE_SUITES}>
+                    {t('label.bundle-suite-plural')}
+                  </Radio.Button>
+                </Radio.Group>
+              </Col>
+              <Col span={8}>
+                <Searchbar
+                  removeMargin
+                  placeholder={t('label.search-entity', {
+                    entity:
+                      subTab === DataQualitySubTabs.TABLE_SUITES
+                        ? t('label.table-suite-plural')
+                        : t('label.bundle-suite-plural'),
+                  })}
+                  searchValue={searchValue}
+                  onSearch={(value) => handleSearchParam(value, 'searchValue')}
+                />
+              </Col>
+            </Row>
+          </div>
+          <Table
+            columns={columns}
+            containerClassName="custom-card-with-table"
+            customPaginationProps={{
+              currentPage,
+              isLoading,
+              pageSize,
+              isNumberBased: true,
+              paging,
+              pagingHandler: handleTestSuitesPageChange,
+              onShowSizeChange: handlePageSizeChange,
+              showPagination,
+            }}
```

Comment:
Let's pull it as variable and pass it instead creating new object on each render